packages/cube/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "barnard59-cube",
+  "version": "2.0.0",
```

Comment:
apparently I'm still confused by versioning  :)

Reply:
This is the current, or initial, version. It could be anything, for sure. And then you added the changeset which bumps major. So with `2.0`, we'd publish `v3` to NPM when this is merged